meta-iot2000-example/conf/distro/poky-iot2000.conf

```diff
@@ -6,19 +6,5 @@ DISTRO_VERSION = "V3.1.17"
 
 BINDINGS_pn-mraa = "python nodejs"
 
-WICVARS += "BBLAYERS \
```

Comment:
So, the `+=` avoid picking up defaults for `WICVARS` that are weak-assigned (`?=`)? The commit message does not make this clear enough yet. Was that list below once the default list wic needed, just replicated into our layer?

Reply:
I think so. appending the value to the `WICVARS`, meaning `WICVARS` has defined values. so would not using the defaults. I would make it more clear in the commit message.

Reply:
Yep. `+=` would overwrite the weak-assigned `?=`, instead of appending.

I checked the commit log of the poky/meta/classes/image_types_wic.bbclass, seems there was not such default value in history.